FirebaseRemoteConfig/Interop/RolloutAssignment.swift

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
```

Comment:
All of the other interops are protocols only. Why is there code here?

Reply:
This object will be shared between Crashlytics and RC to transfer the Rollout related data. We are following the similar similar implementation on Android side. cc @danasilver if you want to add more input on this!